app/services/generate_test_data.rb

```diff
@@ -9,21 +9,34 @@ def generate
     provider.application_choices.map(&:delete)
 
     number_of_candidates.times do
-      application_form = FactoryBot.create(
-        :completed_application_form,
-        application_choices_count: 0,
+      first_name = Faker::Name.first_name
```

Comment:
```suggestion
      first_name = Faker::Name.unique.first_name
```

Reply:
using Faker::Name.unique will stop https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/1119-support-interface-audit-trail-view/spec/services/generate_test_data_spec.rb#L33 from erroring -- the intention there is to test the handling of duplicate codes, not email addresses, and it's good to have unique email addresses in test data, anyway.